blocklist-processing-engine/src/consumers/transfer-consumer.ts

```diff
@@ -0,0 +1,94 @@
+import * as kafka from 'kafka-node';
+import { isBlocked } from "../redis-client/redis-client";
+import { configObj } from '../config/config';
+
+
+const getMSISDN = (message: string): string => {
```

Comment:
I thought I was having to many deja vus, but actually there's some repeated code in multiple files.
It would be nice if all these shared modules would be put inside a `utils` folder where they all can be imported.